### PR TITLE
Add smooth overflow scroll for iPad & iPhone

### PR DESCRIPTION
**Pull request by antibland**
Hi Jason,

I first checked out your project on my iPhone and noticed the scrolling within overflow areas could be smoother. My little commit addresses this.

- Andy

### CODE REVIEW

File: scss/cayman.scss [hunk 1]
```diff
@@ -1,6 +1,10 @@
 $large-breakpoint: 64em;
 $medium-breakpoint: 42em;
 
+@mixin smooth-scroll {
+  -webkit-overflow-scrolling: touch;
+}
```
Comment (jasonlong) [inline scss/cayman.scss]: Since there is just one property here, let's kill the mixin and use `-webkit-overflow-scrolling` directly below.

**Comment by jasonlong** [comment]
Thanks @antibland. I just had the one comment about the mixin. Otherwise, I'm cool trying this out.

**Comment by antibland** [comment]
Thanks. Done.

**Comment by jasonlong** [comment]
:metal:

**Pull request merged**